src/main/webapp/script.js

```diff
@@ -0,0 +1,17 @@
+function init() {
+  new tui.Calendar('#calendar', {
+    defaultView: 'week',
+    useCreationPopup: true,
+    useDetailPopup: true,
+    disableDblClick: true,
+    disableClick: true,
+    isReadOnly: true,
+    scheduleView: ['time'],
+    taskView: false,
+    week: {
+      workweek: true,
+    },
+  });
+}
+
+init();
```

Comment:
also, minor nit: please make sure you end files with a newline (github shows this with a red minus-sign-in-a-circle).